applications/validate_camera_fov.py

```diff
@@ -12,25 +13,45 @@
 
 
 if __name__ == '__main__':
-    '''
-    This is an example application to calculate the FoV of a camera, in this case LST is used.
-    The application prints out the FoV and plots the camera.
-    '''
 
-    print('\nValidating the camera FoV of LST\n')
+    parser = argparse.ArgumentParser(
+        description=(
+            'Calculate the camera FoV of the telescope requested. '
+            'Plot the camera as well, as seen for an observer facing the camera.'
+        )
+    )
+    parser.add_argument(
+        '--tel_type',
+        help='Telescope type (e.g. mst-flashcam, lst)',
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        '--model_version',
+        help='Model version (default=prod4)',
+        type=str,
+        default='prod4'
+    )
+    parser.add_argument(
+        '--site',
+        help='Site (default=South)',
+        type=str,
+        default='south'
+    )
+
+    args = parser.parse_args()
 
-    site = 'south'
-    telescope = 'lst'
-    version = 'prod4'
-    label = 'lst-test'
+    label = '{}-test'.format(args.tel_type.lower())
```

Comment:
All the output files (e.g. from the telescope_model) already contain the telescope type on the name, so it's redundant to add it to the label. I would use a label that only refers to the application itself. For the derive_mirror_rnda, I used derive_rnda.

Reply:
Oh, OK, changed it now to `label = 'validate-FoV'`.

Reply:
Can you explain to me what 'label' is doing / how it is used?

Reply:
It is not relevant for this application. It is used when you compare one model to another. Most common use case is when you want to compare a modified model (lets say with modified random reflection angle) to one in the database (e.g., Prod4). In that case the label will be used in the plot comparing between the two (and in other places I guess, @RaulRPrado can expand I am sure).

Reply:
Label is actually always relevant.
It's used to label all the output files, and the output directory structure. It's suppose to be used by all the modules that produce any kind of output.
There is the option of not giving any label (the date will be used for the output directory name) but it's not a good practice.